samples/javascript_nodejs/49.WebChat-bot/public/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en-US">
+
+<head>
+<title>Web Chat: Full-featured bundle</title>
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<!--
+    This CDN points to the latest official release of Web Chat. If you need to test against Web Chat's latest bits, please refer to pointing to Web Chat's MyGet feed:
+    https://github.com/microsoft/BotFramework-WebChat#how-to-test-with-web-chats-latest-bits
+    -->
+<script src="https://cdn.botframework.com/botframework-webchat/latest/webchat.js"></script>
+<style>
+    html, body { height: 100% }
+    body { margin: 0  }
+
+    #webchat {
+    height: 100%;
```

Comment:
CSS indentation.

Reply:
I indent the 2 lines between the brackets (height: / width:) 2 spaces.